assets/src/edit-story/components/library/common/searchInput.js

```diff
@@ -24,7 +24,7 @@ import { rgba } from 'polished';
 /**
  * Internal dependencies
  */
-import Magnify from './magnify.svg';
+import { ReactComponent as Magnify } from './magnify.svg';
```

Comment:
This icon is not even in the new design, so should be removed.

Reply:
Synced with Sam on the icon, seems like it's better to have it